src/main/java/org/isf/medicalstock/service/MedicalStockIoOperations.java

```diff
@@ -396,23 +326,15 @@ protected boolean updateMedicalOutcomingQuantity(
 	 * @throws OHServiceException if an error occurs during the update.
 	 */
 	@SuppressWarnings("unchecked")
-	protected boolean updateMedicalWardQuantity(
-			Ward ward, 
-			Medical medical, 
-			int quantity,
-			Lot lot) throws OHServiceException
-	{
-		MedicalWard medicalWard = (MedicalWard)medicalStockRepository.findOneWhereCodeAndMedicalAndLot(ward.getCode(), medical.getCode() , lot.getCode());
-				
-		if (medicalWard != null)
-		{			
-			medicalWard.setInQuantity(medicalWard.getInQuantity()+quantity);
+	protected boolean updateMedicalWardQuantity(Ward ward, Medical medical, int quantity, Lot lot) throws OHServiceException {
+		MedicalWard medicalWard = medicalStockRepository.findOneWhereCodeAndMedicalAndLot(ward.getCode(), medical.getCode(), lot.getCode());
+
+		if (medicalWard != null) {
+			medicalWard.setIn_quantity(medicalWard.getIn_quantity() + quantity);
 			medicalStockRepository.save(medicalWard);
-		}
-		else
-		{
+		} else {
 			medicalWard = new MedicalWard(ward, medical, quantity, 0, lot);
-			Double  quan = (double) quantity;
+			Double quan = (double) quantity;
```

Comment:
maybe rename quan -> doubleQuantity, or maybe just boxing it directly in the call?

Reply:
Fixed; by casting the variable in the method call.

Reply:
Thanks!